Dockerfile

```diff
@@ -1,42 +1,62 @@
 ARG PHP_VERSION="8.2"
 ARG DEBIAN_VERSION="bookworm"
 
-FROM php:${PHP_VERSION}-fpm-${DEBIAN_VERSION} as pimcore_php_min
+FROM php:${PHP_VERSION}-fpm-${DEBIAN_VERSION} AS pimcore_php_min
 
-COPY files/build-cleanup.sh /usr/local/bin
-COPY files/build-install.sh /usr/local/bin
-RUN chmod +x /usr/local/bin/build-*
+COPY files/build-*.sh /usr/local/bin
+RUN chmod +x /usr/local/bin/build-*.sh
 
 RUN set -eux; \
+    \
     DPKG_ARCH="$(dpkg --print-architecture)"; \
     echo "deb http://deb.debian.org/debian bookworm-backports main" > /etc/apt/sources.list.d/backports.list; \
     apt-get update; \
     apt-get upgrade -y; \
     \
     # tools used by Pimcore
-    apt-get install -y iproute2 unzip; \
+    apt-get install -y \
+        iproute2 \
+        unzip \
+    ; \
     \
-    # dependencies fór building PHP extensions
+    # dependencies for building PHP extensions
     apt-get install -y \
-        libicu-dev zlib1g-dev libpng-dev libjpeg62-turbo-dev libzip-dev; \
+        libicu-dev \
+        libjpeg62-turbo-dev \
+        libpng-dev \
+        libzip-dev \
+        zlib1g-dev \
+    ; \
     \
-    docker-php-ext-configure pcntl --enable-pcntl; \
     docker-php-ext-configure gd -enable-gd --with-jpeg; \
-    docker-php-ext-install pcntl bcmath pdo_mysql exif zip opcache sockets gd intl; \
+    docker-php-ext-configure pcntl --enable-pcntl; \
+    docker-php-ext-install \
+        bcmath \
+        exif \
+        gd \
+        intl \
+        opcache \
+        pcntl \
+        pdo_mysql \
+        sockets \
+        zip \
+    ; \
+    \
+    build-cleanup.sh; \
     \
     ldconfig /usr/local/lib; \
     \
-    sync;
+    sync
 
-RUN set -eux; build-cleanup.sh;
-
-RUN echo "upload_max_filesize = 100M" >> /usr/local/etc/php/conf.d/20-pimcore.ini; \
-    echo "memory_limit = 256M" >> /usr/local/etc/php/conf.d/20-pimcore.ini; \
-    echo "post_max_size = 100M" >> /usr/local/etc/php/conf.d/20-pimcore.ini
+RUN { \
+        echo "upload_max_filesize = 100M"; \
+        echo "memory_limit = 256M"; \
+        echo "post_max_size = 100M"; \
+    } > /usr/local/etc/php/conf.d/20-pimcore.ini
```

Comment:
Are you sure those three `echo`s do the same as before? Is it possible, than the file was created before and content would be overwritten now?

Reply:
That's impossible, the stage here starts from an official image and the file will be created here for the first time (and it even gets touched once instead of potentially three times).

Reply:
This is a Pimcore-specific file, so if *we* haven't created it before, it won't exist.

Reply:
:+1: